README.md

```diff
@@ -118,6 +118,7 @@ subjective "from more to less liquid control-ly" order.
 
 | Type               | Device family and specific documentation | Notes | MRLV |
 | :--                | :-- | --: | :-: |
+| AIO liquid cooler  | [ASUS ROG RYUJIN II 360](docs/asus-rog-ryujin-guide.md) | <sup>_ep_</sup> | git |
```

Comment:
```suggestion
| AIO liquid cooler  | [ASUS Ryujin II 360](docs/asus-ryujin-guide.md) | <sup>_enp_</sup> | git |
```

- clean up how we name the device (and make it consistent to how we treat other ASUS products): remove quasi-redundant "ROG" and un-shout "Ryujin";
- add missing `n` (new device) note;

---

Please remove "rog" from filenames and update any remaining links/references.

Reply:
> add missing `n` (new device) note

Shouldn't there also be a `n` note in the legend below the table?

Reply:
Oh, please disregard that part. I forgot that the `n` note was removed a while back, and the comment before the table, which I checked, is unfortunately outdated.